src/components/CardGroup.vue

```diff
@@ -0,0 +1,33 @@
+<script setup lang="ts">
+  /************************** 
+  Top level Card container
+  todo: fill out this section
+  ***************************/
+  
+  defineProps<{
+    card_title: string
+    card_description: string
+  }>()
+
+</script>
+
+<template>
+  <article class="card card-group">
+    
+    <header class="card-header">
+      <hgroup>
+        <h2 class="card-title">{{card_title}}</h2>
+        <span class="card-description">{{card_description}}</span>
+      </hgroup>
+    </header>
+
+    <section class="card-body">
+      <slot />
```

Comment:
where is slot defined? i don't see it as a vue component

Reply:
It is a built in... feature.

https://vuejs.org/guide/components/slots.html

Reply:
ahhhh cool. Marko has something similar 👍